examples/kotlin/src/test/groovy/io/github/joke/spockmockable/PersonTest.groovy

```diff
@@ -3,7 +3,7 @@ package io.github.joke.spockmockable
 import org.spockframework.mock.MockUtil
 import spock.lang.Specification
 
-@Mockable([Person, Address])
+@Mockable(packages = "io.github.joke.spockmockable")
```

Comment:
Could you add a separate test file. So we don't loose a test testing lists classes.

Reply:
Of course. I added seperate tests for this feature.